### PR TITLE
Note on difference of sigma_0 for unpolarized and polarized incoming neutrons

### PR DESCRIPTION
**Pull request by astellhorn**

### CODE REVIEW

File: docs/Data_correction_documentary_cleared.ipynb [hunk 1]
```diff
@@ -89,7 +89,7 @@
     "O(\\lambda)=n\\cdot \\sigma_0\\cdot l \\cdot\\lambda\n",
     "$$\n",
     "\n",
-    "with n being the number density of <sup>3</sup>He gas in the cell, $\\sigma_0$ the absorption cross section of 1 $\\mathring A$ neutrons by <sup>3</sup>He ($\\sigma_0 = 2966 \\cdot 10^{−24} cm^2$), and the path length through the cell $l$ [Ref. 1]. As <sup>3</sup>He can be considered to follow the ideal gas law, $n$ can be replaced via $p=n\\cdot k_B\\cdot T$:\n",
+    "with n being the number density of <sup>3</sup>He gas in the cell, $\\sigma_0$ the absorption cross section of 1 $\\mathring A$ neutrons by <sup>3</sup>He ($\\sigma_0 = 2966 \\cdot 10^{−24} cm^2$), and the path length through the cell $l$ [Ref. 1]. Note that 2966 barn at 1 $\\mathring A$ is the unpolarised neutron cross-section. For polarised neutrons, it is a polarization dependent value with 5933 bar at 1 $\\mathring A$ for neutrons with spin anti-parallel to the <sup>3</sup>He and 0 for neutrons with spin parallel to the <sup>3</sup>He. As <sup>3</sup>He can be considered to follow the ideal gas law, $n$ can be replaced via $p=n\\cdot k_B\\cdot T$:\n",
```
Comment (SimonHeybrock) [inline docs/Data_correction_documentary_cleared.ipynb:92]: Do we need to use this value somewhere?

Does the reference contain error bars?

Comment (astellhorn) [reply docs/Data_correction_documentary_cleared.ipynb:92]: The 5933 barn for polarized neutrons will need to be used for adapted workflows e.g. having a supermirror as polarizer (i.e., when the incident neutron beam on the 3He analyzer is polarized). THis is not the case here, but will need to be inserted for an adapted workflow for a different mode of DREAM, and also for SKADI. I am putting together those workflows at the moment.
Btw: sorry I wrote "5933 bar", missing an "n"

Comment (astellhorn) [reply docs/Data_correction_documentary_cleared.ipynb:92]: No, in the literature the marker on the plotted cure (its from 1988, and the  values are only as plot) is bigger than the error bar

Comment (nvaytet) [reply docs/Data_correction_documentary_cleared.ipynb:92]: > the marker [...] is bigger than the error bar

I guess that doesn't always mean we can ignore the error bar? It depends on how you make the plot. If the range on the displayed axes is very large, and you need to make your marker visible to the naked eye, the range the marker itself spans could be significant?

Comment (astellhorn) [reply docs/Data_correction_documentary_cleared.ipynb:92]: Yes I am aware of that and am trying to find more precise information

**Comment by SimonHeybrock** [comment]
Done in #33.

**Pull request closed without merging**